ratis-logservice/src/test/java/org/apache/ratis/logservice/LogServiceReadWriteBase.java

```diff
@@ -166,6 +174,37 @@ public void testLogServiceReadWrite() throws Exception {
         assertEquals(expected, actual);
       }
       testJMXMetrics(logStream);
+      assertEquals(logStream.getState(),State.OPEN);
+
+      //In absense of MetaStateMachine, calling raft directly
```

Comment:
This is a nice test!

Would it be good to make a new unit test which does have both metadata and logserver quorums?